app/main.py

```diff
@@ -1 +1,33 @@
-# write your code here
+class Animal:
+    alive = []
+
+    def __init__(
+        self, name: str,
+        health: int = 100,
+        hidden: bool = False
+    ) -> None:
+        self.name = name
+        self.health = health
+        self.hidden = hidden
+        Animal.alive.append(self)
+
+    def __repr__(self) -> any:
+        return "{" +\
+            f"Name: {self.name}, Health: {self.health}, Hidden: {self.hidden}"\
+               + "}"
```

Comment:
f"{{Name: {self.name}, "
f"Health: {self.health}, "
f"Hidden: {self.hidden}}}" - u can write like this here
or fix idenatation here (up to u)